perceval/backends/core/confluence.py

```diff
@@ -86,11 +86,22 @@ def fetch(self, category=CATEGORY_HISTORICAL_CONTENT, from_date=DEFAULT_DATETIME
 
         from_date = datetime_to_utc(from_date)
 
-        kwargs = {'from_date': from_date}
+        kwargs = {
+            'from_date': from_date
+        }
+
         items = super().fetch(category, **kwargs)
 
         return items
 
+    @staticmethod
+    def ancestors(item, content):
+        ancestors = []
+        if ('ancestors' in content):
+            for ancestor in content['ancestors']:
```

Comment:
to check the test coverage, you could use the package https://coverage.readthedocs.io/en/v4.5.x/